langchain4j-vearch/src/test/java/dev/langchain4j/store/embedding/vearch/VearchEmbeddingStoreIT.java

```diff
@@ -24,15 +16,9 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.data.Percentage.withPercentage;
 
-@Testcontainers
 class VearchEmbeddingStoreIT extends EmbeddingStoreIT {
 
-    @Container
-    static GenericContainer<?> vearch = new GenericContainer<>(DockerImageName.parse("vearch/vearch:3.4.1"))
-            .withExposedPorts(9001, 8817)
-            .withCommand("all")
-            .withCopyFileToContainer(MountableFile.forClasspathResource("config.toml"), "/vearch/config.toml")
-            .waitingFor(Wait.forLogMessage(".*INFO : server pid:1.*\\n", 1));
+    static VearchContainer vearch = new VearchContainer();
```

Comment:
Looks there is an issue in this test from last build:
`Error:    VearchEmbeddingStoreIT>EmbeddingStoreIT.should_add_embedding_with_segment_with_metadata:54 `

Reply:
Done.

The current Vearch implementation uses the old REST API and has too many limitations, Vearch has refactored the API in the new version and supports new features (such as inserting only a part of fields), I think I'll be refactoring `langchain4j-vearch` in the next few weeks!